packages/medusa/src/api/routes/admin/uploads/get-download-url.ts

```diff
@@ -26,18 +26,17 @@ import { validator } from "../../../../utils/validator"
  *     description: OK
  */
 export default async (req, res) => {
-  const validated = await validator(AdminGetUploadsFileDownloadUrlReq, req.body)
-
   const fileService: AbstractFileService<any> = req.scope.resolve("fileService")
 
   const url = await fileService.getPresignedDownloadUrl({
-    fileKey: validated.file_key,
+    fileKey: (req.validatedBody as AdminCreateUploadsFileDownloadUrlReq)
+      .file_key,
   })
 
   res.status(200).send({ download_url: url })
 }
 
-class AdminGetUploadsFileDownloadUrlReq {
+export class AdminCreateUploadsFileDownloadUrlReq {
```

Comment:
todo: Follow payload naming convention
```suggestion
export class AdminPostUploadsDownloadUrlReq {
```